tsconfig.build.json

```diff
@@ -66,7 +66,7 @@
 
     /* Advanced Options */
     "skipLibCheck": true /* Skip type checking of declaration files. */,
-    "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
+    // "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
     "resolveJsonModule": true
   }
 }
```

Comment:
I think github's syntax highlighting for json just doesnt understand comments in json. You can see the same thing here for example: https://github.com/mui/material-ui/blob/4b735de27e84a0221f42530d281f6d718ee8a7e1/packages/mui-utils/tsconfig.build.json#L4

Reply:
I see you're referring to the curly brace specifically... same issue though it looks like. It's fine locally. As a sanity check I added an extra brace and that makes the build command fail. Which is not happening here in CI so I think we're good.

Reply:
I removed the unused option for good measure but it doesnt change anything. looks like that's just the way it is on main too https://github.com/interledger/rafiki/blob/main/tsconfig.build.json

This is actually kinda weird, since some other tsconfigs with comments show the same way: https://github.com/yurtsiv/mermaid/blob/5d4d7c5fbfa21bc9d4974613fb7052144df336e0/todo-fix-root-level-tsconfig.json

while others do not: https://github.com/Ejden/vuetify/blob/1b8a04b9d936ba98d9e37ce711fd49aef53b1213/tsconfig.json

In the ones with red syntax highlighting the comma is at the end of the comment instead of the value. Not sure if thats the cause but prettier forces that on our end and it's just a display issue on github after all.

Reply:
After you pointed it out it really bugged me that I wasn't entirely sure why this happens 😆